github-dork.py

```diff
@@ -8,7 +8,9 @@
 import feedparser
 from copy import copy
 from sys import stderr
+from common import post_to_slack, get_logger
```

Comment:
This doesn't seem like something relevant to this project. If you intend to add slack support, you should do it in a way that allows for configuration and works in multiple environments.

Reply:
Actually, now that you've changed this, I'm not really sure what the goal is here.